module/validation/seal_validator.go

```diff
@@ -257,11 +257,11 @@ func (s *sealValidator) validateSeal(seal *flow.Seal, incorporatedResult *flow.I
 
 	executionResultID := executionResult.ID()
 	for _, chunk := range executionResult.Chunks {
```

Comment:
```suggestion
	// if a valid seal doesn't require any approval, we could just skip
	if s.requiredApprovalsForSealVerification == 0 {
		return nil
	}
	
	for _, chunk := range executionResult.Chunks {
```


And we could also remove [this case](https://github.com/onflow/flow-go/pull/285/files#diff-c17ba09a7fa83290fab3f113c7397b8c8902bec5b134539822e531e6e5e062feR215), because in that case, there won't be any error